### ISSUE TITLE
Prefix additional commands with `st`

### ISSUE THREAD

**Issue by aviau** [syncthing/syncthing#3360]
I would like to ship the additional Syncthing binaries in Debian.

Mostly the following:
 - discosrv
 - relaysrv

The names of these binaries are pretty general. It would be great if they were moved into the *syncthing namespace*.

I suggest the following:
 - syncthing-discosrv
 - syncthing-relaysrv

Cheers

**Comment by calmh** [comment]
We have the `st` prefix on a lot of other internal binaries. That would probably be acceptable.

**Comment by calmh** [comment]
For anyone engaging in this, it needs at minimum
- renaming the dirs in `cmd/`
- updating the build.go script appropriately
- updating the [documentation repo](https://github.com/syncthing/docs) (all references)
- updating the [release tools](https://github.com/syncthing/release-tools)
- renaming the repos where we publish releases? maybe not required in step one
- renaming the jobs on the build server again maybe not required immediately

It's a fair amount of busywork so don't jump into it without due consideration :)

**Comment by aviau** [comment]
Will you merge my PR if I rename things to ```syncthing-```? I think that is it less risky for the future and will provide better protection against possible namespace clashes with other projects/tools.

I also think that it would make it easier to find for users as they will only have to look for ```syncthing```.

If you disagree, I will still take the time to open a PR to rename things to ```st-```, as it is better than nothing.

**Comment by calmh** [comment]
No, there's a clear pattern here:

```
jb@syno:~/s/g/s/syncthing $ ls cmd/
discosrv/	stbench/	stdisco/	stfileinfo/	stgenfiles/	stsigtool/	stwatchfile/	todos/
relaysrv/	stcompdirs/	stevents/	stfinddevice/	stindex/	stvanity/	syncthing/
```

("todos" is never exposed to the outside and should really live in script/)

A new, longer, different prefix seems counterproductive to me.

**Comment by aviau** [comment]
Fair enough, I'll start working on this!

**Comment by calmh** [comment]
Closed by #3371
